RELEASE.md

```diff
@@ -0,0 +1,107 @@
+# Che Theia release process
+
+#### Make the release changes
+
+1. Create a branch for the release, e.g. `7.7.0`
```

Comment:
just a notice, but in other repos, branches are named `7.7.x`
https://github.com/eclipse/che/tree/7.7.x

`7.7.0` is often used as a `tag name`

Reply:
yeah, I saw that
I've just decided to keep it as I received from the former IDE2 team so as not to accidentally break something

Reply:
sure, just maybe try for the upcoming releases to match other repositories

Reply:
I'll try

Reply:
proposed a PR https://github.com/eclipse/che-theia/pull/630